### PR TITLE
UPDATE documentation for GenAi IBM and OPENAI

### PR DESCRIPTION
**Pull request by cvivekkartik**

### CODE REVIEW

File: docs/UCV/README.md [hunk 1]
```diff
@@ -408,6 +410,22 @@ SonarQube is an open source tool suite to measure and analyze the quality of sou
 | :---: | :---: | :---: | :---: |
 |[All Plugins](../index.md)|[Top](#contents)|[Readme](testcomplete-ucv/README.md)|[0]()|
 
+## GenAi-summary-release-ibm
+
+The GenAI Summary Release - ibm plugin for DevOps Velocity is designed to streamline and enhance your workflow by automatically generating concise business value summaries for candidate releases. Leveraging cutting-edge generative AI technology, this plugin interprets issue data to provide meaningful summaries that can be directly used in your business communications.
```
Comment (madhavbk) [inline docs/UCV/README.md]: use IBM all over, dont use ibm as its an entity

File: docs/UCV/ucv-ext-release-summary-openai/usage.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+### Install - Offline
+1.  Pull release summary docker image
+2.  Navigate to the `settings/integrations` page of Velocity  and select the `Installed` tab.
+2.  Click the `Load Plugin` button near the top of the page, and enter in `[RELEASE SUMMARY IMAGE NAME]:[LATEST_TAG]`.
+3.  Select `Submit` and the plugin will be installed. You will then see the `GenAI Summary Release - OpenAI` in the list of plugins (or if already present, the new uploaded version). Skip the requested configuration.
+
+### Install - Online
+
+1. Navigate to the `settings/integrations` page of Velocity  and select the `Available` tab.
+2. Select `GenAI Summary Releaser` plugin and click `Install` action.
```
Comment (madhavbk) [inline docs/UCV/ucv-ext-release-summary-openai/usage.md]: plugin name spelling mistake 'Releaser' ( there is 'r' at the end )

File: docs/UCV/ucv-ext-release-summary-openai/usage.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    curl -i -X POST \
+        -H "Content-Type:application/json" \
+        -d \
+        '{
+            "vsmid" : "6641e92d735e31769cb2f7cc",
+            "releaseid" : "669548dc6d0a6b58e6a0add4",
+            "targetStageName" : "PROD",
+            "currentStageName" : "DEV"
+        }' \
+    'https://10.14.82.75/reporting-consumer/pluginEndpoint/669e12a21e0e8a001b70ec35/generate'
```
Comment (madhavbk) [inline docs/UCV/ucv-ext-release-summary-openai/usage.md]: Usually dont quote IP addresses rather quote <domain>/... etc

**Review by madhavbk — CHANGES_REQUESTED** [review]

File: docs/UCV/README.md (hunk 1 above)
Comment (cvivekkartik) [reply docs/UCV/README.md]: Requested changes are done

File: docs/UCV/ucv-ext-release-summary-openai/usage.md (hunk 2 above)
Comment (cvivekkartik) [reply docs/UCV/ucv-ext-release-summary-openai/usage.md]: Requested changes are done

File: docs/UCV/ucv-ext-release-summary-openai/usage.md (hunk 3 above)
Comment (cvivekkartik) [reply docs/UCV/ucv-ext-release-summary-openai/usage.md]: Requested changes are done

**Review by madhavbk — APPROVED** [review]

**Pull request merged**